packages/core/src/api/firmware/firmwareBase.ts

```
@@ -0,0 +1,378 @@
+  checkDeviceToBootloader(connectId: string | undefined) {
+    this.checkPromise = createDeferred();
+    const env = DataManager.getSettings('env');
+    const isBleReconnect = connectId && DataManager.isBleConnect(env);
+
+    Log.log('FirmwareUpdateV2 [checkDeviceToBootloader] isBleReconnect: ', isBleReconnect);
+
+    // check device goto bootloader mode
+    let isFirstCheck = true;
+    let checkCount = 0;
+    // eslint-disable-next-line prefer-const
+    let timeoutTimer: ReturnType<typeof setTimeout> | undefined;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'timeoutTimer' is only assigned here.


(lint/style/useConst)